update-motd.d/99-livepatch

```diff
@@ -59,7 +60,10 @@ if ! echo "$PATH" | grep -qE "(^|:)/snap/bin/?(:|$)"; then
 fi
 
 service_name="livepatch"
-ua_status=$($UA status 2>&1)
+# if there is no cache file yet (the cron job hasn't run yet), create a
+# fresh one. We also want to ignore an empty cache file.
+[ -s "$UA_STATUS_CACHE" ] || "$UA" status > "$UA_STATUS_CACHE" 2>&1
```

Comment:
why are you redirecting stderr to stdin? this means any error message would end up in the cache

Reply:
Because it's what the former `$($UA status 2>&1)` did. I guess I should also do that in the cron job (or not do it, if we decide to change that here as well).

Reply:
I don't think you should do it, because if for some reason some stderr text goes in, it could be parsed in place of what's expected